templates/consumer.init.erb

```diff
@@ -21,7 +21,7 @@ start() {
     else
         exec $DAEMON $DAEMON_OPTS >> /var/log/kafka/consumer.log 2>&1 &
         sleep 2
-        PID=`ps axww | grep -E '[k]afka-console-consumer' | awk '{print $1}'`
+        PID=`ps axww | grep -E '[k]afka.tools.ConsoleConsumer' | awk '{print $1}'`
```

Comment:
that's not so much a regular expression, as it simply assures that grep itself won't show up in the grepped output

pgrep does the same thing as the above, but it does so while not needing two pipes, and three forks of three different binaries.